sdk/monitor/Azure.Monitor.OpenTelemetry/tests/Azure.Monitor.OpenTelemetry.Demo/Program.cs

```diff
@@ -15,6 +17,12 @@
     o.ConnectionString = "InstrumentationKey=00000000-0000-0000-0000-000000000000";
 });
 
+// To customize sampling, Set ApplicationInsightsSampler to desired sampling ratio and
+// configure with OpenTelemetryTracerProvider.
+// Please note that ConfigureOpenTelemetryTracerProvider should be called after
+// builder.Services.AddAzureMonitorOpenTelemetry().
+builder.Services.ConfigureOpenTelemetryTracerProvider((sp, builder) => builder.SetSampler(new ApplicationInsightsSampler(0.9F)));
```

Comment:
could we check if the sampler can expose settings, and support DI based config, so that this extra config here could be simplified to services.Configure<ApplicationInsightsSamplerOptions>(opt => opt.Ratio = .09) ?

Reply:
This require changes to the `ApplicationInsightsSampler`. We plan to get this in next version after bringing changes to `ApplicationInsightsSampler`